lib/test.js

```diff
@@ -222,13 +222,13 @@ class Test {
 
 		const deferRecording = this.metadata.inline;
 		this.deferredSnapshotRecordings = [];
-		this.compareWithSnapshot = ({expected, id, message}) => {
+		this.compareWithSnapshot = ({expected, message}) => {
 			this.snapshotCount++;
 
 			// TODO: In a breaking change, reject non-undefined, falsy IDs and messages.
```

Comment:
Yes, empty messages should result in an improper use failure. Just for this assertion, I don't think it matters much for the others. The intent should be that the message is used in the snapshot report, and passing an empty string for that is nonsensical.